fuzz/fsfuzzer.py

```diff
@@ -81,7 +81,7 @@ def setUp(self):
         self._args = self.params.get('fstype', default='')
         self._fsfuzz = os.path.abspath(os.path.join('.', "fsfuzz"))
         fs_sup = process.system_output('%s %s' % (self._fsfuzz, ' --help'))
-        match = re.search(br'%s' % self._args, fs_sup, re.M | re.I)
+        match = re.search(br'%s' % self._args, fs_sup.decode(), re.M | re.I)
```

Comment:
`b` must be removed here?

Reply:
No - I have tested all combinations and we need to keep the br'%s' for search to be properly done.

Reply:
can you paste a sample output with matches the search?

Reply:
1) Currently Reproduced traceback from: /usr/local/lib/python3.7/site-packages/avocado_framework-78.0-py3.7.egg/avocado/core/test.py:854
Traceback (most recent call last):
  File "/home/fedora/avocado-fvt-wrapper/tests/avocado-misc-tests/fuzz/fsfuzzer.py", line 84, in setUp match = re.search(br'%s' % self._args, fs_sup, re.M | re.I)
TypeError: %b requires a bytes-like object, or an object that implements __bytes__, not 'str'
ERROR 1-tests/avocado-misc-tests/fuzz/fsfuzzer.py:Fsfuzzer.test -> TestSetupFail: %b requires a bytes-like object, or an object that implements __bytes__, not 'str'

2) If I Add a debug message, and get rid of 'b' 
[stdout] DEBUG: self.args= fs_sup=b'./fsfuzz cramfs|ext2|ext3|ext4|swap|btrfs|iso9660|msdos|squashfs|vfat|xfs'
[stdout] 


Reproduced traceback from: /usr/local/lib/python3.7/site-packages/avocado_framework-78.0-py3.7.egg/avocado/core/test.py:854
Traceback (most recent call last):
  File "/home/fedora/avocado-fvt-wrapper/tests/avocado-misc-tests/fuzz/fsfuzzer.py", line 85, in setUp
    match = re.search(r'%s' % self._args, fs_sup, re.M | re.I)
  File "/usr/lib64/python3.7/re.py", line 183, in search
    return _compile(pattern, flags).search(string)
TypeError: cannot use a string pattern on a bytes-like object

3) If I add decode to fs_sup then It sounds to work so you are right, my appologize for not having it right first time - thanks for the review.
[stdout] DEBUG: self.args= fs_sup=b'./fsfuzz cramfs|ext2|ext3|ext4|swap|btrfs|iso9660|msdos|squashfs|vfat|xfs' <class 'bytes'>
[stdout]